lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/FileBasedAccessControl.java

```diff
@@ -118,7 +118,7 @@ public FileBasedAccessControl(String catalogName, FileBasedAccessControlConfig c
     @Override
     public RoleSupport getRoleSupport()
     {
-        return RoleSupport.CONNECTOR;
+        return RoleSupport.GLOBAL;
```

Comment:
Why not define roles for file based connector access control? Why do we use system roles here?

Reply:
Connector roles are not only used with HMS. They are other access control that expressed as Trino connector access control that have roles.